addons/l10n_hk/data/l10n_hk_chart_data.xml

```diff
@@ -0,0 +1,522 @@
+        <field name="code">5229</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_expenses" />
+        <field name="tag_ids" eval="[(6,0,[ref('account.account_tag_operating')])]" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_5231" model="account.account.template">
+        <field name="name">Disposal</field>
+        <field name="code">5231</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_expenses" />
+        <field name="tag_ids" eval="[(6,0,[ref('account.account_tag_operating')])]" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_5234" model="account.account.template">
+        <field name="name">Repair and Maintenance</field>
+        <field name="code">5234</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_expenses" />
+        <field name="tag_ids" eval="[(6,0,[ref('account.account_tag_operating')])]" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_5235" model="account.account.template">
+        <field name="name">Advertising</field>
+        <field name="code">5235</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_expenses" />
+        <field name="tag_ids" eval="[(6,0,[ref('account.account_tag_operating')])]" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+    <record id="l10n_hk_5240" model="account.account.template">
+        <field name="name">Foreign Exchange Loss</field>
+        <field name="code">5240</field>
+        <field name="reconcile" eval="False" />
+        <field name="user_type_id" ref="account.data_account_type_expenses" />
+        <field name="tag_ids" eval="[(6,0,[ref('account.account_tag_operating')])]" />
+        <field name="chart_template_id" ref="l10n_hk_chart_template" />
+    </record>
+
+    <!-- Account Chart Template -->
+    <record id="l10n_hk_chart_template" model="account.chart.template">
+        <field name="name">Hong Kong - Chart of Accounts</field>
+        <field name="transfer_account_id" ref="l10n_hk_transfer" />
+        <field name="property_account_receivable_id" ref="l10n_hk_1240" />
+        <field name="property_account_payable_id" ref="l10n_hk_2211" />
+        <field name="property_account_income_categ_id" ref="l10n_hk_41" />
+        <field name="property_account_expense_categ_id" ref="l10n_hk_51" />
+        <field name="expense_currency_exchange_account_id" ref="l10n_hk_5240" />
+        <field name="income_currency_exchange_account_id" ref="l10n_hk_4240" />
+        <field name="use_anglo_saxon" eval="True" />
```

Comment:
Why do you still keep this True?

Reply:
since the Hong Kongish accounting is supposed to be anglo saxon, we'll keep it True... even if for invoicing/small size companies they don't need it, it is just normal to have it set.